templates/motor-insurance/inputs/selected-payment-term-label.js

```diff
@@ -0,0 +1,10 @@
+function label(key) {
+    return {
```

Comment:
This hash can be in the module scope to avoid creating it more than once, so:

```javascript
const labels = {
    'annual-card': 'Annual',
    'monthly-card': 'Monthly by credit/debit card',
    'monthly-directdebit': 'Monthly by direct debit',
    'monthly-directdebit-card': 'Monthly by direct debit'
};

export default function label(key) {
    return labels[key];
}
```
Does there need to be a default label?